rbac/models.py

```
@@ -0,0 +1,1015 @@
+            password,
+            **extra_fields
+        )
+
+
+class UnifiedUser(AbstractBaseUser, PermissionsMixin):
+    """
+    مدل کاربر یکپارچه برای بیماران و پزشکان
+    این مدل به عنوان کاربر مادر عمل می‌کند
+    """
+    
+    # شناسه‌ها
```

Ruff (0.12.2)

Comment contains ambiguous `ه` (ARABIC LETTER HEH). Did you mean `o` (LATIN SMALL LETTER O)?

(RUF003)